CDP4Orm/Dao/Resolve/ResolveDao.cs

```diff
@@ -86,8 +86,8 @@ private static IEnumerable<ResolveInfo> ReadSiteDirectoryThing(NpgsqlTransaction
 
             // get all Thing 'concepts' that are newer then passed in revision
             sqlBuilder.Append(
-                "SELECT \"ValueTypeSet\"-> 'ClassKind' as \"TypeInfo\", \"Iid\"").Append(
-                " FROM \"{0}\".\"Thing_View\"").Append(
+                "SELECT \"ValueTypeDictionary\"-> 'ClassKind' as \"TypeInfo\", \"Iid\"").Append(
```

Comment:
may i request we refactor this so it becomes more readable. using the numbered arguments and a combi of string builder and string.format can be improved

Reply:
refactored